doc/apiv3-documentation.apib

```diff
@@ -3507,6 +3818,8 @@ Gets a list of users that can watch the work package.
 
     **Required permission:** add work package watchers
 
+    *Note that you will only receive this error, if you are at least allowed to see the corresponding project.*
```

Comment:
I bet this is :spaghetti: Copy pasta again. This should probably read "work package" instead of "project".